hitch/content.md

```diff
@@ -0,0 +1,63 @@
+# What is Hitch?
+
+[Hitch](https://hitch-tls.org/) is a *libev-based* high performance *SSL/TLS proxy* by [Varnish Software](https://varnish-software.com). It is specifically built to terminate TLS connections at high scale and forwards unencrypted HTTP traffic to Varnish or any other HTTP backend.
+
+# How to use this image
+
+Running a Hitch Docker container can be done by using the following command:
+
+```console
+docker run --name=hitch -p 443:443 hitch:latest
+```
+
+This container will expose port `443`, which is required for HTTPS traffic.
+
+## Configuration file and extra options
+
+Without any argument, the container will run `hitch --config=/etc/hitch/hitch.conf`. You can mount your own configuration file to replace the default one:
+
+```console
+docker run -v /path/to/your/config/file:/etc/hitch/hitch.conf hitch
+```
+
+You can also change the path of the configuration file by setting the `HITCH_CONFIG_FILE` environment variable. You can set it to an empty string to disable the configuration file altogether.
```

Comment:
Ah @gquintard, this is the behavior my version of the script lost for you -- however, wouldn't it be easier/more intuitive to suggest users change the command to `hitch` to disable the default behavior?  ie:

```console
$ docker run ... %%IMAGE%% hitch
```

Reply:
agreed

```suggestion
You can also change the path of the configuration file by setting the `HITCH_CONFIG_FILE` environment variable.
```